website/templates/project/view_file.mako

```diff
@@ -41,9 +34,12 @@
 
     <!-- Menu toggle closed -->
     <div class="panel panel-default osf-panel-show text-center reset-height pointer"  style="display: none">
-      <div class="panel-heading">
-        <i class="fa fa-angle-right"></i>
-      </div>
+        <div class="row tb-header-row">
```

Comment:
Minor NP but could we just fix the the parent's indentation and make it in line with [the html style guidelines](http://cosdev.readthedocs.org/en/latest/style_guides/html_css.html#html-and-css) since we are already there.